indico/modules/events/templates/display/common/_manage_button.html

```diff
@@ -30,13 +31,15 @@
         {% set note_item = item.session if item_type == 'SESSION_BLOCK' else item %}
         {{ _render_note_editor(note_item) }}
     {%- elif opt == 'notes_compile' -%}
+        {% set title = _('Compile Minutes')%}
         <div class="manage-notes-container"
              data-get-note-url="{{ url_for('event_notes.compile', item) }}"
              data-api-url="{{ url_for('event_notes.api', item) }}"
              {% if item.can_manage_attachments(session.user) %}
                  data-image-upload-url="{{ url_for('attachments.upload_editor', item) }}"
              {% endif %}
-             data-title="{% trans %}Compile minutes{% endtrans %}"></div>
+             data-modal-title="{{ title }}"
+             data-title="{{ title }}"></div>
```

Comment:
Is this actually used for anything? Feels like a leftover from before this was reactified.

Reply:
nevermind it's just a weird name for the content of the dropdown item that opens it...